experimental/quarkus/image/Dockerfile-stack

```diff
@@ -1,27 +1,28 @@
 # Dockerfile for building the stack
-FROM quay.io/quarkus/centos-quarkus-maven:19.0.2
+FROM quay.io/quarkus/centos-quarkus-maven:19.1.1
 
 COPY ./project /project
 COPY ./config /config
+
 WORKDIR /project
 
-USER root
-RUN mkdir -p /mvn/repository
-RUN mvn -Dmaven.repo.local=/mvn/repository install dependency:go-offline -DskipTests
+RUN mvn install dependency:go-offline -DskipTests
+
+ENV APPSODY_USER_RUN_AS_LOCAL=true
 
-ENV APPSODY_MOUNTS=/:/project/user-app
-ENV APPSODY_DEPS=/mvn/repository
+ENV APPSODY_MOUNTS=".:/project/user-app/;~/.m2/repository:/home/$(id -u)/.m2/repository"
```

Comment:
I think id -u will give u the uid. Should we use $(id -un) as that would give you the name of the user?

Reply:
good catch, your right it should be `-un`, found a small merge error in version numbers so fixed that as well.